tokens/test/erc-721/Rarible.test.js

```diff
@@ -322,6 +322,83 @@ contract("ERC721Rarible", accounts => {
     );
   });
 
+  it("mint and transfer with minter access control", async () => {
+    const minter = accounts[1];
+    let transferTo = accounts[2];
+
+    const tokenId = minter + "b00000000000000000000001";
+    const tokenURI = "//uri";
+
+    await token.enableMinterAccessControl({from: tokenOwner});
+    assert.equal(await token.minterAccessControlEnabled(), true);
+
+    await expectThrow(
+      token.mintAndTransfer([tokenId, tokenURI, creators([minter]), [], [zeroWord]], transferTo, {from: minter})
+    );
+
+    await token.grantMinter(minter, {from: tokenOwner})
+    assert.equal(await token.isValidMinter(minter), true);
+    assert.equal(await token.isValidMinter(transferTo), false);
+
+    await token.mintAndTransfer([tokenId, tokenURI, creators([minter]), [], [zeroWord]], transferTo, {from: minter})
+    assert.equal(await token.ownerOf(tokenId), transferTo);
+  });
+
+  it("mint and transfer with minter access control and minter signature", async () => {
+    const minter = accounts[1];
+    let transferTo = accounts[2];
+
+    const tokenId = minter + "b00000000000000000000001";
+    const tokenURI = "//uri";
+
+    await token.enableMinterAccessControl({from: tokenOwner});
+    assert.equal(await token.minterAccessControlEnabled(), true);
+
+    const signature = await getSignature(tokenId, tokenURI, creators([minter]), [], minter);
+
+    let whiteListProxy = accounts[5];
+    await token.setDefaultApproval(whiteListProxy, true, {from: tokenOwner});
```

Comment:
also in the last version of rarible tokens (where isPrivate variable appears) there is no setDefaultApproval() method, so you probably need to take that into consideration

Reply:
removed in commit https://github.com/rarible/protocol-contracts/pull/105/commits/6a5e213b9ed8f38f84d2397dcf61771d038c4b8e